### PR TITLE
Format code to Google Java Style Guide standards.

### PR DESCRIPTION
**Pull request by zongweil**
Google-originated open source projects should follow the [Google Style Guide](https://github.com/google/styleguide). The Java style guide can be found [here](https://google.github.io/styleguide/javaguide.html).

### CODE REVIEW

File: java/com/google/openlocationcode/OpenLocationCode.java [hunk 1]
```diff
@@ -7,462 +7,548 @@
...
+      return eastLongitude.doubleValue();
     }
+  }
 
 
-    /** The state of the OpenLocationCode. */
-    private final String code;
+  /**
+   * The state of the OpenLocationCode.
+   */
```
Comment (jirisemecky) [inline java/com/google/openlocationcode/OpenLocationCode.java]: FYI, the one-line javadoc comments are also allowed, and for one line definition probably more readable.

**Comment by jirisemecky** [comment]
Looks good to me.

**Comment by zongweil** [comment]
I added another commit to make single-line JavaDoc comments where possible (had to adjust an option on my IntelliJ formatter and make the changes manually). I also had to change the email address on my git commits to pass the CLA checks (hence the "outdated diff" comment).

**Comment by jirisemecky** [comment]
Great, thanks!

**Comment by drinckes** [comment]
Awesome!

**Review by Yankii001 — COMMENTED** [review]
explore

**Comment by Yankii001** [comment]
> explore

update upaload

**Pull request merged**